File: beginner/pagina01/1006/1006.java
import java.io.BufferedReader;
import java.io.InputStreamReader;
import java.io.IOException;

public class Main {
	public static void main(String[] args) throws IOException {
		InputStreamReader ir = new InputStreamReader(System.in);
		BufferedReader in = new BufferedReader(ir); 

		double a = Double.parseDouble(in.readLine());
		double b = Double.parseDouble(in.readLine());
		double c = Double.parseDouble(in.readLine());
		double average = ((a * 2) + (b * 3) + (c * 5)) / 10;

		System.out.printf("MEDIA = %.1f\n", average);
	}
}

File: beginner/pagina01/1006/1006.cs
using System;
using System.Globalization;

public class URI 
{
    public static void Main(string[] args)
    {

        double a = double.Parse(Console.ReadLine());
        double b = double.Parse(Console.ReadLine());
        double c = double.Parse(Console.ReadLine());

        double average = (a * 2 + b * 3 + c * 5) / 10.0;

        Console.WriteLine("MEDIA = {0}", average.ToString("F1", CultureInfo.InvariantCulture));
    }
}
